backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -249,14 +249,13 @@ public SqlSchema createSchema(String name, String description) {
     return getSchema(name);
   }
 
-  private static void validateSchemaIdentifierIsUnique(SchemaMetadata metadata, Database db) {
+  private static void validateNameIsLowercaseUnique(SchemaMetadata metadata, Database db) {
     for (String name : db.getSchemaNames()) {
-      if (!metadata.getName().equals(name)
-          && metadata.getIdentifier().equals(new SchemaMetadata(name).getIdentifier())) {
+      if (!metadata.getName().equals(name) && metadata.getName().equalsIgnoreCase(name)) {
         throw new MolgenisException(
             String.format(
-                "Cannot create/alter schema because name resolves to same identifier: '%s' has same identifier as '%s' (both resolve to identifier '%s')",
-                metadata.getName(), name, metadata.getIdentifier()));
+                "Cannot create/alter schema because name is equal to existing schema in different case: '%s' is same as  '%s' (both resolve to '%s')",
```

Comment:
Checking for duplicate names is nice, but it would be nice to have this fix (feat?) in a different PR so it can be tested in a vacuum.